ebu_tt_live/node/test/test_delay_unit.py

```diff
@@ -1,9 +1,31 @@
 from unittest import TestCase
-from ebu_tt_live.node.delay import RetimingDelayNode
+from mock import MagicMock
+from ebu_tt_live.node.delay import RetimingDelayNode, BufferDelayNode
 
 
 class TestRetimingDelayNode(TestCase):
 
     def test_process_document(self):
 
-        pass
+        carriage = MagicMock()
+        reference_clock = MagicMock()
+        delay = 2
+        document_sequence = 'TestSequence'
+        node = RetimingDelayNode('delay_node', carriage, reference_clock, delay, document_sequence)
+        document = MagicMock()
+        node.process_document(document)
+        carriage.emit_document.assert_called_with(document)
```

Comment:
I don't understand what's going on here: does this not require a delay parameter? Sorry this is my ignorance!